client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content.tsx

```diff
@@ -177,15 +177,16 @@ export default function SiteStatusContent( {
 	let content;
 
 	if ( type === 'stats' ) {
+		const { total: totalViews, trend: viewsTrend } = rows.stats.data.views;
 		return (
 			<>
 				<Gridicon
-					icon={ `arrow-${ statTrend }` }
+					icon={ `arrow-${ viewsTrend }` }
 					size={ 18 }
 					className={ `sites-overview__icon-${ statTrend }` }
```

Comment:
Even though I like this kinda approach, we should avoid doing this for the sake of searchability.

I would suggest creating a function that returns the icon based on the trend.

```
function getStatTrendIcon(trend) {
   if(trend === "up") {
     return "arrow-up"
     ...
  }
}
```
Do you think class names will be needed here? If yes, we could try to include that in the same function.

Reply:
I can skip using an extra class by targeting the `.gridicons` class.

Reply:
It makes sense for the searchability aspect side of things.